concepts/60 Themes and Styles/05 Predefined Themes/50 Apply a Theme/05 Using SCSS.md

```diff
@@ -0,0 +1,47 @@
+You can import the theme bundle from the `bundles` folder from the `devextreme` package in the root `.scss` file:
+
+    /* @use 'devextreme/scss/bundles/dx.theme_name.scss';  */
+    @use 'devextreme/scss/bundles/dx.carmine.scss';
+
+Or you can import individual files for the whole theme. The example below imports files of the `Material Blue Light Compact` theme
```

Comment:
> Or you can import individual files for **the** whole theme.

Again, I see a definite article. Why? We don't have to speak of a specific theme, or even **the same** indefinite theme we mentioned earlier on. Because the things we speak of are valid for **any** possible theme. Therefore, there is no reason to interpret this case as definite.

```suggestion
Alternatively, you can import individual files that comprise a theme. The example below imports files from the `Material Blue Light Compact` theme:
```